app/Http/Controllers/Api/TilesController.php

```
@@ -0,0 +1,118 @@
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\File;
+
+class TilesController extends Controller
+{
+    /**
+     * Serve tile files from storage/data/tiles directory
+     */
+    public function serveTile(Request $request, $layer, $z, $x, $y)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (Unused Code Rules)

(UnusedFormalParameter)